subsys/bluetooth/controller/ll_sw/ull_scan.c

```diff
@@ -102,6 +103,10 @@ u8_t ll_scan_enable(u8_t enable)
 	}
 #endif
 
+#if defined(CONFIG_BT_CTRL_TX_PWR_DYNAMIC_CTRL)
```

Comment:
This needs to be moved to `ull_scan_enable` below!

Reply:
Done [here](https://github.com/zephyrproject-rtos/zephyr/pull/19779/commits/b8c7a15362e46dbcc1238e7bf79694779119a062#diff-5fde7b773bf177bd42dddfd1715205a1R178)